static/src/hbs/_scripts.hbs

```diff
@@ -1 +1,2 @@
-<script src='{{ assetPath "js/site.js" }}'></script>
+{% load staticfiles %}
```

Comment:
Correct. The reason is that when this template is rendered by Django it won't have leading newlines.